ci/jenkins/README.md

```diff
@@ -85,15 +85,26 @@ DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
 ./ci/jenkins/test.sh --testcase networkpolicy --registry ${DOCKER_REGISTRY}
 ```
 
-* windows e2e: e2e tests in a Windows setup with Docker runtime.
+* windows e2e: e2e tests in a Windows setup with Containerd runtime.
 
 ```shell
 #!/bin/bash
 DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
 ./ci/jenkins/test.sh --testcase windows-e2e --registry ${DOCKER_REGISTRY}
 ```
 
-* windows conformance: community tests on Windows cluster with Docker runtime, focusing on "Conformance|sig-windows" and
+* windows ovs e2e: e2e tests in a Windows setup with Containerd runtime and OVS running
+  inside container.
+
+```shell
+#!/bin/bash
+DOCKER_REGISTRY="$(head -n1 ci/docker-registry)"
+./ci/jenkins/test.sh --testcase windows-e2e --registry ${DOCKER_REGISTRY}
```

Comment:
May I know why the testcase is `windows-e2e` in for containerd runtime?

Reply:
changed to windows-ovs-e2e

Reply:
@NamanAg30 , testcase is still windows-e2e for containerd ?